### ISSUE TITLE
Modal example with no header

### ISSUE THREAD

**Issue by jgiardino** [patternfly/patternfly-next#445]
Add an example of a modal that has no header.

In this case, `aria-label` would include the label for the modal, instead of having `aria-labelledby` reference the header `id`.

The accessibility documentation should be updated to include `aria-label` with an explanation that it would be used when no header is included in the modal.

**Comment by jgiardino** [comment]
@mcarrano @andresgalante 
This example was discussed at the a11y review meeting last week. I just want to confirm that this is a case we support and that this example should be added.

**Comment by jgiardino** [comment]
Related to this, a question came up about where the Close button goes when there isn't a header. Do we still have the header section, or does it move to the body section?

**Comment by mattnolting** [comment]
Was using https://bulma.io/documentation/components/modal/ as an example. @andresgalante stated the modal _should_ have a header, but I think Bulma's approach is valid. They distinguish between a simple modal and card style modal, wherein the simple modal presents an article, with a thumbnail, etc.

**Comment by mcarrano** [comment]
@jgiardino @mattnolting While I can see this as a useful alternative to the current modal, we have not supported it in the past and I don't feel like there is an immediate use case that requires a modal without a header. Was there a reason why this was added/requested?
